packages/apps-config/src/api/spec/subsocial.ts

```diff
@@ -1,9 +1,12 @@
 // Copyright 2017-2022 @polkadot/apps-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
+// IMPORTANT
+//
 // We only load the definitions here explicitly - if we try to go via
-//   import { types } from '@subsocial/types/substrate';
-// we end up with multiple version of types/API
-import definitions from '@subsocial/types/substrate/interfaces/subsocial/definitions';
+//   import { types } from '@subsocial/types';
```

Comment:
if import is not required , please remove it

Reply:
About this, I need more confidence to remove it. So, I'd like to keep this change.